src/engine/Resources/Sound.ts

```diff
@@ -274,8 +276,15 @@ export class Sound implements ILoadable, IAudio {
 
    /**
     * Play the sound, returns a promise that resolves when the sound is done playing
+    * Valid string inputs include:
+    * -'linearUp'
+    * -'linearDown'
+    * -'ease'
+    * -'ease-in'
+    * -'ease-out'
+    * -'ease-in-out'
     */
-   public play(volume?: string): Promise<boolean> {
+   public play(data?: string | number | IBezierPoints): Promise<boolean> {
```

Comment:
I guess I'm confused on whether it's better to just pass in a string:

```
sound.playWithProfile('linearUp')
```

or to create Types and then assign those to Strings. I'm picturing that the method call would look identical except for the quotes:
```
sound.playWithProfile(linearUp)
```